include/ibm/locks.hpp

```diff
@@ -317,7 +253,7 @@ inline RcAcquireLock Lock::acquireLock(const LockRequests& lockRequestStructure)
 
     if (status)
     {
-        BMCWEB_LOG_DEBUG << "There is a conflict within itself";
+        BMCWEB_LOG_ERROR << "There is a conflict within itself";
```

Comment:
this change is out of scope for this PR.why was this made ERROR ?

Reply:
This change is to log a trace if there is a lock conflict